pygments/lexers/c_like.py

```diff
@@ -1,9 +1,7 @@
 """
     pygments.lexers.c_like
     ~~~~~~~~~~~~~~~~~~~~~~
-
```

Comment:
Where does this change and the ones below come from? Is your editor stripping empty lines automatically? Please add these empty lines back; some of them are required by reST syntax.